src/components/HeaderHome.jsx

```diff
@@ -28,7 +28,7 @@ const HeaderHome = () => (
             <p className={styles.headerDescription}>
               <div style={{ display: 'flex', flexDirection: 'column' }}>
                 {siteDescription}
-                <div>
+                <div style={{ marginTop: '10px' }}>
```

Comment:
please no inline styles unless there really is no other way. If you need styling in this file there is already a CSS modules file for that so this should be continued to be used instead of inline styles.